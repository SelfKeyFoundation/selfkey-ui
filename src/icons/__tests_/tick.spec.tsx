import * as React from 'react';
import { create } from 'react-test-renderer';
import TickIcon from '../tick';

describe('Tick Icon', () => {
	it('should show Tick Icon', () => {
		const component = create(<TickIcon />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
