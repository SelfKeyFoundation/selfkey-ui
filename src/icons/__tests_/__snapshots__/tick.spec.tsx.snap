// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tick Icon should show Tick Icon 1`] = `
<svg
  aria-hidden="true"
  className="MuiSvgIcon-root Component-root-0-2-1"
  focusable="false"
  viewBox="0 0 11 9"
>
  <path
    d="M4.16 5.613l-1.54-1.54A.995.995 0 0 0 1.21 4.07l-.107.106a1.003 1.003 0 0 0 .003 1.411L3.088 7.57c.049.115.12.222.214.316a.995.995 0 0 0 1.411.003l5.41-5.41a.994.994 0 0 0-.002-1.412.995.995 0 0 0-1.412-.003L4.161 5.613z"
    fill="#7896B9"
    fillRule="evenodd"
  />
</svg>
`;
